class/defaults.yml

```diff
@@ -2,3 +2,25 @@ parameters:
   floodgate:
     =_metadata: {}
     namespace: syn-floodgate
+    namespaceLabels: {}
+
+    imageDay: "1"
```

Comment:
Maybe `imageBuildDay` would make it more clear what this parameter does? I had to look up the docs immediately...